.github/workflows/latest_dependency_checker.yml

```diff
@@ -25,7 +25,7 @@ jobs:
         source venv_core/bin/activate
         python -m pip install --upgrade pip
         python -m pip install -e .
```

Comment:
I think this line could be removed as `python -m pip install .[test]` should also install the base library and its requirements in addition to the test requirements.